servicepulse/host-config.md

```diff
@@ -24,13 +24,17 @@ The default value for ImagePath is:
 Change the value after `--url=` and restart the service.
 
 
-## Changing the ServiceControl URL
+## Default connection to ServiceControl and ServiceControl Monitoring
 
-ServicePulse is configured to connect to the ServiceControl REST API. To specify the URL to connect do  the following
+ServicePulse will by default, attempt to connect to ServiceControl and ServiceControl Monitoring using the URLs `http://localhost:33333/api/` and `http://localhost:33633/`, respectively.
 
-Note: Ensure version 1.3 or above is being used.
+## Configuring connections via the ServicePulse UI
 
- * Go to the installation directory for ServicePulse. Typically this is `C:\Program Files (x86)\Particular Software\ServicePulse\`
- * Go to the `app\js` directory and edit `app.constants.js`
- * Delete any `*.js` files in the `app` directory
- * Change the value of the url after `service_control_url`
\ No newline at end of file
+The default connections can easily be overriden via the Connections configuration screen in ServicePulse:
+
+![Connections configuration](images/connections-configuration.png 'width=500')
+
+When entering the connection URL to ServiceControl and ServiceControl Monitoring, the full URLs including the port number and scheme needs to be entered in the Connections screen. Connection configuration is persisted only locally and is read in the following order:
+
+1. **Query string in browser address bar URL** - allows easily bookmarking and sharing of connection details.
```

Comment:
@mauroservienti @HEskandari @WilliamBZA I was led here while reviewing https://github.com/Particular/docs.particular.net/pull/4394. I'm not sure I understand this. Does this mean that, when I switch to another ServiceControl instance, the address to ServiceControl is then embedded in the URL for ServicePulse?

Reply:
Correct.